agent/reconnectingpty/reconnectingpty.go

```diff
@@ -64,13 +66,20 @@ func New(ctx context.Context, logger slog.Logger, execer agentexec.Execer, cmd *
 	// runs) but in CI screen often incorrectly claims the session name does not
 	// exist even though screen -list shows it.  For now, restrict screen to
 	// Linux.
-	backendType := "buffered"
+	autoBackendType := "buffered"
```

Comment:
Suggestion: can we call this `plain`? Otherwise it feels like we're leaking implementation details.

Reply:
I'll do this as a follow-up refactor 👍